.tav.yml

```diff
@@ -528,6 +528,18 @@ aws-sdk:
     - node test/instrumentation/modules/@aws-sdk/client-s3.test.js
   node: '>=14'
 
+
+'@aws-sdk/client-sns':
+  # Maintenance note: This should be updated periodically using:
+  #   node ./dev-utils/tav-versions.js @aws-sdk/client-sns ">=3.15.0 <4"
+  #
+  # Test v3.15.0, every N=40 of 205 releases, and current latest.
+  versions: '3.15.0 || 3.53.0 || 3.154.0 || 3.218.0 || 3.294.0 || 3.360.0 || 3.370.0 || >3.370.0 <4'
```

Comment:
nit: We will want to update this to the same version as client-s3 (`"@aws-sdk/client-s3": "^3.379.1",`) to reduce duplicate installs of all the transitive deps.

Reply:
nope, they don't release all packages for every version.